src/jobs/scamDomainUpdateTimers.ts

```diff
@@ -0,0 +1,23 @@
+import fetch from 'node-fetch';
+import { logger } from '../logger';
+import { parentPort } from 'node:worker_threads';
+import Redis from 'ioredis';
+
+const redis = new Redis(process.env.REDISHOST);
+const list = await fetch(process.env.SCAM_DOMAIN_URL!).then((r) => r.json());
+const before = await redis.scard('scamdomains');
+
+await redis.sadd('scamdomains', ...list);
```

Comment:
You may want to make `scandomains` into a string constant for reuse elsewhere.

Reply:
fully agree, i do keys as constants or functions in my own projects, however yuudachi seems to be using hardcoded redis keys at the moment, which I went with here as well

Reply:
I feel like this is consistent with the current codebase, but generally a refactor to use functions/exported constants for redis keys might be in order